application-workloads/visualstudio/az-400-dev-env/README.md

```diff
@@ -0,0 +1,30 @@
+# AZ-400 Lab Environment, with all necessary tools/softwares pre-installed
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/application-workloads/visualstudio/az-400-dev-env/PublicLastTestDate.svg)
+![Azure Public Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/application-workloads/visualstudio/az-400-dev-env/PublicDeployment.svg)
+
+![Azure US Gov Last Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/application-workloads/visualstudio/az-400-dev-env/FairfaxLastTestDate.svg)
+![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/application-workloads/visualstudio/az-400-dev-env/FairfaxDeployment.svg)
+
+![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/application-workloads/visualstudio/az-400-dev-env/BestPracticeResult.svg)
+
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/application-workloads/visualstudio/az-400-dev-env/CredScanResult.svg)
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fapplication-workloads%2Fvisualstudio%2Faz-400-dev-env%2Fazuredeploy.json)
+[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fapplication-workloads%2Fvisualstudio%2Faz-400-dev-env%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fapplication-workloads%2Fvisualstudio%2Faz-400-dev-env%2Fazuredeploy.json)   
+
+
+AZ-400 (Azure DevOps Expert) Certification program had lots of Labs for participants. I have observed, many of my participants did not had ready environment to practice labs. I have created this ARM Template to make it easier for participants.
```

Comment:
fix typos

AZ-400 (Azure DevOps Expert) Certification program had lots of labs for participants. I have observed, many of my participants did not have an environment ready to practice labs. I have created this ARM Template to make it easier for participants.

Reply:
replaced with:
Template for setting up dev-environment for AZ-400 (Azure DevOps Certification) Labs.